README.md

```diff
@@ -118,15 +117,15 @@ if ( 2 > 1 ) {
 }
 `);
 console.log(result);
-// prints { data: "Hello World!", txn_time: "2022-10-31T00:00:00Z" } // txn_time will vary!
+// prints { data: "Hello World!", txn_ts: 1667174400000000 } // txn_ts will vary!
 ```
 
 Note that with TypeScript you can apply a type parameter to your result. So querying with a type \<T\> you'll get a shape back like:
 
 ```(typescript)
 {
   data: T,
-  txn_time: string
+  txn_ts: string
```

Comment:
Should this be number?

```suggestion
  txn_ts: number
```

Reply:
It comes over the wire as such

![image](https://user-images.githubusercontent.com/7998974/221903426-38dac12c-bb43-450a-88d7-c0fac191b4d7.png)

Reply:
I think this is okay for JS to parse as a number because it will be a very very long time before a unix epoch is outside of safe js integers